qcfractal/interface/client.py

```diff
@@ -240,18 +242,46 @@ def add_molecules(self, mol_list: List[Molecule], full_return: bool = False) ->
 
     ### Keywords section
 
-    def get_keywords(self, opt_list):
+    def get_keywords(self, id: List[str], full_return: bool=False) -> 'List[KeywordSet]':
+        """Obtains KeywordSets from the server using keyword ids.
 
-        body = KeywordGETBody(meta={}, data=opt_list)
+        Parameters
+        ----------
+        id : List[str]
+            A list of ids to query.
+        full_return : bool, optional
+            Optionally returns the full request result including the ``meta`` field.
+
+        Returns
+        -------
+        List[KeywordSet]
+            The requested KeywordSet objects.
+        """
+        body = KeywordGETBody(meta={}, data=id)
         r = self._request("get", "keyword", data=body.json())
         r = KeywordGETResponse.parse_raw(r.text)
 
-        return r.data
+        if full_return:
```

Comment:
I rather prefer the `return_full` pattern established by https://github.com/MolSSI/QCElemental/blob/master/qcelemental/molparse/from_string.py#L17 and `return_wfn` but little matters.